depthai_helpers/calibration_utils.py

```diff
@@ -209,16 +241,38 @@ def analyze_charuco(self, images, resize_img_func = None):
         # SUB PIXEL CORNER DETECTION CRITERION
         criteria = (cv2.TERM_CRITERIA_EPS +
                     cv2.TERM_CRITERIA_MAX_ITER, 10000, 0.00001)
+        count = 0
         for im in images:
             if self.traceLevel == 2:
```

Comment:
Where is the traceLevel initialized. you seem to have removed it and never initialized it from the function Class level. If ran. this code might break. 
Did you check the execution ?

Reply:
It is initialized in calibrate.py as an argument and then passed to stereo_calib in line 824 in calibrate.py.
I checked on OAK-D-PRO and it worked fine.